astropy/coordinates/solar_system.py

```diff
@@ -124,128 +185,124 @@ def get_body_barycentric(time, body):
 
     Notes
     -----
-    """ +_JPL_EPHEM_NOTE
+    """ + _EPHEMERIS_NOTE
```

Comment:
It turns out this way of doing it erases the docstrings (and has some other issues).  I think I've got it working, though, so I'll issue a PR against your branch with the fix for that.